stm/asyncTaskQueue.ml

```diff
@@ -146,9 +146,10 @@ module Make(T : Task) () = struct
           set_slave_opt tl
       in
       let args =
-        Array.of_list (set_slave_opt (List.tl (Array.to_list Sys.argv))) in
+        let wselect = "--kind=" ^ !T.name in
```

Comment:
IIUC T.name is eg "proofworker", but argument parsing expects "proof"
How can that work?

Reply:
Fixed, testing locally.